### DISCUSSION TITLE
Connecting to internet on Windows XP VM

### DISCUSSION THREAD

**Discussion by rmart1n-dev** [utmapp/UTM#5580] [Windows]
**The question**  
I'm most likely being dumb, but I would like to connect to the Internet on a Windows XP VM.

**UTM/Mac Configuration**  
* UTM Version: 4.3.5
* macOS Version: Ventura 13.5
* Mac Chip (Intel, M1, ...): M1

***I don't know if a debug log is needed for this. If needed, will provide.***

I'm using the default Windows XP .utm profile provided at the website ([https://mac.getutm.app/gallery/windows-xp](url))

**Comment by pan93412** [comment]
Using Windows XP SP3 (`zh-tw_windows_xp_professional_with_service_pack_3_x86_cd_vl_x14-74140.iso`). Same profile. Same device (4.3.5, Ventura 13.5, M1). Driver installed manually ([spice-guest-tools-latest.exe](https://www.spice-space.org/download/windows/spice-guest-tools/spice-guest-tools-latest.exe)). The network works as intended.

<img width="1136" alt="image" src="https://github.com/utmapp/UTM/assets/28441561/098c4ad1-f6b5-4136-9fe4-36cd6cab6462">

**Comment by osy** [comment]
You probably are trying to visit a website that uses HTTPS. Windows XP doesn't have any certificates installed that works on the 2023 web. You may want to download Firefox.
